docs/design/server_side_charts.md

```diff
@@ -0,0 +1,44 @@
+# Supporting charts on the server side
+As discussed in #367, the server side components manager and resourcifier have not yet been refactored, as of this writing, to work with [the new chart format](docs/design/chart_format.md). Presumably, expandybird should not be affected.
```

Comment:
I had a look at expandybird last weekend.  I could not find any API-level documentation for it, but I managed to figure most of it out by reading the code and poking a running instance of it.  One thing we should probably do, though, is write the API doc for it.